packages/x-date-pickers/src/locales/ar.ts

```diff
@@ -0,0 +1,97 @@
+import { PickersLocaleText } from './utils/pickersLocaleTextApi';
+import { getPickersLocalization } from './utils/getPickersLocalization';
+import { TimeViewWithMeridiem } from '../internals/models';
+
+// maps TimeView to its translation
+const views: Record<TimeViewWithMeridiem, string> = {
+  hours: 'ساعات',
```

Comment:
These are used below, and instruct the user to choose the unit.
it's better to have them singular instead (ex. "اختر الساعة")
```suggestion
  hours: 'الساعة',
```